msteams-platform/graph-api/meeting-transcripts/meeting-insights.md

```diff
@@ -0,0 +1,171 @@
+
+## Prerequisites
+
+* You must enable the transcription or recording for the meeting for which the insights are to be generated. You can programmatically set a meeting to autotranscribe or autorecord using the [update onlineMeeting API](/graph/api/onlinemeeting-update?view=graph-rest-1.0&preserve-view=true&tabs=http) or directly through the [meeting options](/microsoftteams/manage-meeting-recording-options#record-and-transcribe-automatically).
+
+* The Meeting AI Insights API only works with delegated permissions and hence requires a token from a signed-in user to be passed in the call. The signed-in user must have a [Microsoft 365 Copilot license](/copilot/microsoft-365/microsoft-365-copilot-licensing) and access to the meeting’s transcript file.
+
+## Fetch meeting insights
+
+To fetch the insights of a particular meeting, follow these steps:
+
+1. If you don’t have the meeting identifier (`id`), call online meeting API with the `JoinWebUrl` property to retrieve the `id`. For more information, see [retrieve an online meeting by JoinWebUrl](/graph/api/onlinemeeting-get?view=graph-rest-1.0&preserve-view=true&tabs=http#example-3-retrieve-an-online-meeting-by-joinweburl).
+
+1. Each transcript event of the meeting creates an associated [AI insight object](/graph/api/resources/callaiinsight?view=graph-rest-beta&preserve-view=true). Use the [List AI Insights API](/graph/api/onlinemeeting-list-aiinsights?view=graph-rest-beta&preserve-view=true) to fetch all the AI insight objects related to the meeting and use the included metadata in the response to select the relevant AI insight object for your scenario. Here's an example request and response:
+
+    **Request**
+
+    ```http
+    GET /copilot/users/{userId}/onlineMeetings/{onlineMeetingId}/aiInsights
+    ```
+
+    **Response**
+
+    ```json
+    HTTP/1.1 200 OK
+    Content-Type: application/json
+    
+    {
+      "@odata.context": "https://graph.microsoft.com/beta/$metadata#copilot/users('000000000-0000-0000-0000-000000000000')/onlineMeetings('00000...')/aiInsights",
+      "@odata.count": 1,
+      "value": [
+        {
+          "id": "VjEj...",
+          "callId": "af630fe0-04d3-4559-8cf9-91fe45e36296",
+          "contentCorrelationId": "bc842d7a-2f6e-4b18-a1c7-73ef91d5c8e3",
+          "createdDateTime": "2025-05-09T08:17:10.7261294Z",
+          "endDateTime": "2025-05-09T08:17:10.7261294Z"
+        }
+      ]
+    }
+    ```
+
+    | Property | Description |
+    | --- | --- |
+    | `id` | A unique identifier for the generated AI insight object. |
+    | `callId` | A unique identifier for the [call](/graph/api/resources/call?view=graph-rest-1.0&preserve-view=true) during which this insight is generated. |
```

Comment:
I don't think this is accurate. It is not related to the PMA call leg id that is provided in that Call object.

Reply:
We should state it as "Unique identifier of the call." as other resources say.